code/src/layouts/app-banner/messages/en.json

```diff
@@ -1,4 +1,9 @@
 {
   "appBanner.header": "Incredible Prices on All Your Favorite Items",
-  "appBanner.paragraphfirst": "Wellcome to our store! Here you will find the best gadgets from around the world, and you can choose the perfect product for you."
+  "appBanner.paragraphfirst": "Welcome to our store! Here you will find the best gadgets from around the world, and you can choose the perfect product for you.",
+  "subintro.firstParagraph": "Curb-side\npickup",
```

Comment:
You should move translations for subintro to separate translations folder for subintro

Reply:
Where should this folder be located? Inside the component folder?

Reply:
inside the layout folder. Check #73 PR
![image](https://github.com/idx-academy/spa-orders/assets/118522791/d6e4d799-fbb9-456c-a5d5-033279b09517)

Reply:
Okay, done